lib/correios/cep/version.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 module Correios
   module CEP
-    VERSION = '0.6.4'
+    VERSION = '0.6.5'
```

Comment:
This file should not be changed in feature branches. See the instructions for contributing: https://github.com/prodis/correios-cep#contributing-to-correios-cep.